src/routes/transactions/mappers/common/transfer-direction.helper.ts

```diff
@@ -3,13 +3,17 @@ import { TransferDirection } from '../../entities/transfer-transaction-info.enti
 
 @Injectable()
 export class TransferDirectionHelper {
```

Comment:
This class doesn't seem to provide a lot of value given that the only functionality that it has is represented by a function.

What do you think about just having the function (and no class)?

Reply:
Yes, good point. We can remove a lot of boilerplate code and injection logic that way, thanks!

Changed in [b53dc23](https://github.com/5afe/safe-client-gateway-nest/pull/206/commits/b53dc2327a87d90e6c7c67f3e0b8f7fa41ad7167)